unabridged-works-volume-3/original/12-jews-natural-spiritual.adoc

```diff
@@ -849,6 +882,7 @@ one living body, consisting of a living head, and living members;
 for the same life and pure heavenly nature which is in the head,
 is communicated by him to the members.
 
+[.offset]
```

Comment:
I wasn't sure about this either, but it looked a little funny in the PDF only on one line.

Reply:
yeah, i think i'm good with this 👍